config/importmap.rb

```diff
@@ -2,8 +2,8 @@
 
 # Pin npm packages by running ./bin/importmap
 
-pin 'application', preload: true
-pin '@hotwired/turbo-rails', to: 'turbo.min.js', preload: true
-pin '@hotwired/stimulus', to: 'stimulus.min.js', preload: true
-pin '@hotwired/stimulus-loading', to: 'stimulus-loading.js', preload: true
-pin_all_from 'app/javascript/controllers', under: 'controllers'
+# pin 'application', preload: true
+# pin '@hotwired/turbo-rails', to: 'turbo.min.js', preload: true
+# pin '@hotwired/stimulus', to: 'stimulus.min.js', preload: true
+# pin '@hotwired/stimulus-loading', to: 'stimulus-loading.js', preload: true
+# pin_all_from 'app/javascript/controllers', under: 'controllers'
```

Comment:
Because we are using esbuild to bundle our JavaScript modules, we do not need to use the import maps feature in our Rails application. Instead, we can use esbuild to bundle ourr JavaScript modules and generate a single output file that can be included in our Rails application.
We keep the code until everything work well because I might misunderstanding something.